Fedora/notepadqq.spec

```diff
@@ -21,27 +21,37 @@ Requires:		qt5-qtsvg
 
 
 %description
-An advanced basic text editor.
-Features:
-* Syntax Highlighting
-* Color Themes
-* Code-Collapse
-* Macro-recorder
-
+A qt text editor for developers, with advanced tools, but remaining simple.
+It has Syntax highlighting, themes, macros and more
```

Comment:
Macros are not yet supported! They should be removed from the description.
In addition, the "S" of "Syntax highlighting" should probably be lower case :)